reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/BlockingAwait.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.reaktive.completable
+
+import com.badoo.reaktive.maybe.blockingGet
+
+fun Completable.blockingAwait() {
```

Comment:
RxJava spec returns exception here, if it was thrown. But I find it kind of strange.

Reply:
Where did you see this?
![image](https://user-images.githubusercontent.com/26204457/86486878-6bb2bd80-bd54-11ea-9547-8e08966ae97e.png)

Reply:
Oh, it was `blockingGet`, nvm.